docs/cloudsync_mqtt.md

```diff
@@ -162,3 +163,12 @@ From another terminal/post,it is recommended make a curl get command as follows
 ```
 curl --location --request GET 'http://<ip where edge-orchestration is running>:56001/api/v1/orchestration/cloudsyncmgr/getsubscribedata/<topicname>/<appid>'
 ```
+# 6. DataStructures Used
+### a. URLData 
+This is a hashmap of the type <string,[]string> to store the client ids (appids) mapped to the Url of the broker to which the client are connected. For any first time entry of url in the hashmap, a connection is established between the broker and homeedge. For any subequent entry for the same url, the client id is appended to the list of client ids and same connection object is used for communication. In case of any client calling disconnect or wants to terminate the connection with broker, the client id can be removed from the list of client ids. Incase of one single client for any url calling the disconnect or wanting to terminate the connection, then the connection with the broker is closed and url entry with its key is removed from the Hashmap.
```

Comment:
```suggestion
This is a hashmap of the type <string,[]string> to store the client ids (appids) mapped to the Url of the broker to which the client are connected. For any first time entry of url in the hashmap, a connection is established between the broker and Home Edge. For any subsequent entry for the same url, the client id is appended to the list of client ids and same connection object is used for communication. In case of any client calling disconnect or wants to terminate the connection with broker, the client id can be removed from the list of client ids. In case of one single client for any url calling the disconnect or wanting to terminate the connection, then the connection with the broker is closed and url entry with its key is removed from the Hashmap.
```